addons/auth_signup/models/res_users.py

```diff
@@ -344,9 +344,8 @@ def create(self, vals_list):
                     users_with_email.partner_id.with_context(create_user=True).signup_cancel()
         return users
 
-    def copy(self, default=None):
-        self.ensure_one()
+    def copy_data(self, default=None):
         if not default or not default.get('email'):
```

Comment:
add below code above this line
default = dict(default or {})

Reply:
Yes, it is working perfectly fine with `default=None`.
Sorry I missed it.
Thanks!